Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_ServiceNow_Compliance_Ticket.yml

```diff
@@ -0,0 +1,861 @@
+      - "22"
+    scriptarguments:
+      ComplianceIssueDescription:
+        complex:
+          root: inputs.ComplianceIssueDescription
+      ComplianceIssueID:
+        complex:
+          root: inputs.ComplianceIssueID
+      ComplianceIssueSeverity:
+        complex:
+          root: inputs.ComplianceIssueSeverity
+      Filter:
+        complex:
+          root: filter
+      TicketID:
+        complex:
+          root: Ticket
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: Ticket.Active
+                iscontext: true
+              right:
+                value:
+                  simple: "true"
+          accessor: ID
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 430,
+          "y": 920
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: 3d37b8ab-8ae0-453b-8f40-7fdc79b4d284
+    type: playbook
+    task:
+      id: 3d37b8ab-8ae0-453b-8f40-7fdc79b4d284
+      version: -1
+      name: Prisma Cloud Compute - ServiceNow Ticket (XLSX)
+      playbookName: Prisma Cloud Compute - ServiceNow Ticket (XLSX)
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      ComplianceIssueDescription:
+        complex:
+          root: inputs.ComplianceIssueDescription
+      ComplianceIssueID:
+        complex:
+          root: inputs.ComplianceIssueID
+      ComplianceIssueSeverity:
+        complex:
+          root: inputs.ComplianceIssueSeverity
+      Filter:
+        complex:
+          root: filter
+      TicketID:
+        complex:
+          root: Ticket
+          accessor: ID
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 870,
+          "y": 920
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: cb821891-ede4-4d78-8fdc-8b70c1b0820f
+    type: regular
+    task:
+      id: cb821891-ede4-4d78-8fdc-8b70c1b0820f
+      version: -1
+      name: Delete "Ticket" duplicates
+      description: |-
+        The reason for this task is to make sure that the next task will get a single value for each key.
+        Since this playbook may contain several "Ticket" objects which all of them are with the same ID, depends on the scenario, this task will make sure only one of them will be used in the next task.
```

Comment:
```suggestion
        Since this playbook may contain several "Ticket" objects with the same ID, depending on the scenario, this task will make sure only one of them will be used in the next task.
```